### ISSUE TITLE
Plugin parameters

### ISSUE THREAD

**Issue by lpointet** [uncatcrea/wp-appkit#93]
It seems that some plugins may need to get some params, for example with a declaration like the following:
```
<gap:plugin name="nl.x-services.plugins.launchmyapp" version="3.2.0">
   <param name="URL_SCHEME" value="customurlscheme" />
</gap:plugin>
```

Currently, this type of feature isn't supported by AppKit, and params are simply removed from the export zip.

**Comment by lpointet** [comment]
@mleroi Please review my commit (https://github.com/lpointet/wp-app-kit/commit/d09b61ba6316f88d023215e57779dc9ac0189fbf) to validate what I did about it.

**Comment by mleroi** [comment]
Yeah! very well handled, that's nice! :D
Just tested it both ways (in the BO "plugins" textarea, and in the $default_plugins in code) and it works really well!

I just commited a very small adjustment detail that I noticed reading the code (but is really of no importance at all :p ) : added an "s" to the plugin_data array key ($plugin_data['params']) as this is an array of params. If this is ok for you.
